week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,12 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(number) {
+  const charge = 0.2;
+  const cal = number * charge;
+  const total = number + cal;
+  return total;
```

Comment:
Really easy to understand the solution, only part is cal is not clear what it means, something like taxAmount will be better